lookupapi/urls.py

```diff
@@ -57,6 +57,7 @@
 urlpatterns = [
     path('attributes/people', views.PersonFetchAttributes.as_view(), name='person-attributes'),
     path('people', views.PersonList.as_view(), name='person-list'),
+    path('people/self', views.PersonSelf.as_view(), name='person-self'),
```

Comment:
Could we have "people/self/token"? I.e. a scheme of "self" and an identifier of "token"? Then a) it fits into the existing URL structure and b) it allows other ways of identifying "self" in future.

Reply:
for example?

Reply:
As discussed: Use scheme == "token" and identifier == "self".